packages/flutter/test/material/tooltip_theme_test.dart

```diff
@@ -616,9 +620,9 @@ void main() {
     _ensureTooltipVisible(key);
     await tester.pump(const Duration(seconds: 2)); // faded in, show timer started (and at 0.0)
 
-    final TextStyle textStyle = tester.widget<Text>(find.text(tooltipText)).style!;
+    final TextStyle textStyle = tester.widget<RichText>(_findTooltipText(tooltipText)).text.style!;
     expect(textStyle.color, Colors.orange);
-    expect(textStyle.fontFamily, null);
+    expect(textStyle.fontFamily, 'Roboto');
```

Comment:
Using `Text.rich` changed a few test assertions from `null` to `'Roboto'`. Is that acceptable? @chunhtai

Reply:
the finder should find widget<Text> instead of widget<RichText>

Reply:
ah nice! Was able to cleanup all the tests with this change.